posthog/api/named_query.py

```diff
@@ -210,7 +209,10 @@ def run(self, request: Request, name=None, *args, **kwargs) -> Response:
                     if variable.get("code_name", "") == code_name:
                         variable["value"] = value
 
-            # Build QueryRequest
+            query_overrides = data.query_override or {}
```

Comment:
let's allow it only for the insights, let's make the API contract narrow, so we don't have support things we don't want in the future